test/e2e-test/pages/overview-page/overview-page.ts

```diff
@@ -0,0 +1,34 @@
+import { paths } from '../../../../app/paths';
+import i18n from '../../../../locale/en.json';
+
+const config = require('config');
+
+const testUrl = config.get('testUrl');
+
+module.exports = {
+  overviewPage(I) {
+    When(/^I visit the overview page$/, async () => {
+      await I.amOnPage(testUrl + paths.overview);
+    });
+
+    When(/^I should see the 'do this next section' for "([^"]*)"$/, (appealStage) => {
+      switch (appealStage) {
+        case 'New - Appeal started': {
+          I.see(i18n.pages.overviewPage.doThisNext.toDo, '//*[@id="doThisNextTitle"]/h2[1]');
+          I.seeInSource(`<p class="govuk-body">${i18n.pages.overviewPage.doThisNext.appealStarted.fewQuestions}</p>`);
+          break;
+        }
+        case 'Saved - Appeal started': {
+          I.see(i18n.pages.overviewPage.doThisNext.toDo, '//*[@id="doThisNextTitle"]/h2[1]');
+          I.seeInSource(`<p class="govuk-body">${i18n.pages.overviewPage.doThisNext.appealStarted.fewQuestions}</p>`);
+          break;
+        }
+        case 'Appeal submitted': {
+          I.see(i18n.pages.overviewPage.doThisNext.nothingToDo, '//*[@id="doThisNextTitle"]/h2[1]');
+          I.seeInSource(`<p class="govuk-body">${i18n.pages.overviewPage.doThisNext.appealSubmitted.detailsSent}</p>`);
+          break;
+        }
```

Comment:
Any reason not to have 3 methods 
When(/^I should see the 'do this next section' for 'New - Appeal started'$/, (appealStage) => {
When(/^I should see the 'do this next section' for 'Saved - Appeal started'$/, (appealStage) => {
When(/^I should see the 'do this next section' for 'Appeal submitted'$/, (appealStage) => {

Reply:
Will split on the next commit thank you